LICENSE

```diff
@@ -7,3 +9,19 @@ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 THE SOFTWARE.
+
+------------------------------------------------------------------------------
```

Comment:
You might want to add an acknowledgements note here and head the license note with the name of the project so it doesn't look like The Android Open Source Project also has copyright on our SDK

>Acknowledgments
>Portions of this Button Software may utilize the following copyrighted material, the use of which is hereby acknowledged.
>
> DiskLruCache